Sources/Sentry/Public/SentryScope.h

```diff
@@ -103,10 +104,11 @@ NS_SWIFT_NAME(Scope)
 /**
  * Adds the Scope to the event
  */
-- (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
-                          maxBreadcrumb:(NSUInteger)maxBreadcrumbs;
+- (nullable SentryEvent *)applyToEvent:(SentryEvent *)event
+                         maxBreadcrumb:(NSUInteger)maxBreadcrumbs
+    NS_SWIFT_NAME(applyTo(event:maxBreadcrumbs:));
 
-- (void)applyToSession:(SentrySession *)session;
+- (void)applyToSession:(SentrySession *)session NS_SWIFT_NAME(applyTo(session:));
```

Comment:
`m`: I think those two don't need to be public. I would make them internal by moving them to SentryScope+Private.h